services/book.services.js

```diff
@@ -0,0 +1,32 @@
+const logger = require("../logs/logger");
+const db = require("../models");
+const ERROR_MSG = require("../utils/const");
+const Book = db.books;
+
+function createNewBook(title, isbn, author) {
+  return Book.create({
+    title,
+    isbn,
+    author,
+  })
+    .then((book) => {
+      return book;
+    })
+    .catch((err) => {
+      logger.error(ERROR_MSG, err);
+      next(err);
+    });
+}
+
+function fetchAllBooks() {
+  return Book.findAll()
```

Comment:
It's not a good practice to have an endpoint to return all records in the database at once since the record won't always just like 10 to 20 records. Imagine you have 10000000 records. This endpoint won't be useful.

Hence, make it paginated. Allow user set page number and page size.

In addition, introduce filter such that they can filter records with author, isbn, createdBy, isAvailable etc.